lib/message_builder.rb

```diff
@@ -94,11 +98,85 @@ def no_pull_requests
     render "no_pull_requests"
   end
 
+  def prs_count_message
+    if @all_prs_count > 1
+      "You have #{@all_prs_count} Dependabot PRs open on the following apps:"
+    else
+      "You have 1 Dependabot PR open on the following app:"
+    end
+  end
```

Comment:
I'd move this into the `templates/dependapanda.text.erb` as an inline bit of templating, i.e.

```erb
<% if @all_prs_count > 1 %>
...
<% else %>
...
```

Better to keep snippets of text in the view where possible and just make the message_builder about passing the necessary data to form the view (i.e. repo count etc).